.azure-pipelines/idChangeValidator.yaml

```diff
@@ -0,0 +1,12 @@
+jobs:
+- job: "IdChangeValidation"
```

Comment:
Maybe change the job name and yaml file name to "DetectionsValidations" or sommething like that, as we talked.